5.5/Dockerfile

```diff
@@ -25,9 +25,18 @@ RUN yum install -y \
     yum clean all -y && \
     mkdir /tmp/sessions
 
-ENV PATH $PATH:/opt/rh/php55/root/usr/bin
+RUN sed -i -e 's/^Listen 80/Listen 0.0.0.0:8080/' \
```

Comment:
I wonder if it's worth putting this in an external script that we ADD and RUN.  what do you think?  (you can use the contrib dir I believe)